### ISSUE TITLE
Error in outlier_detection for IFU data

### ISSUE THREAD

**Issue by hbushouse** [spacetelescope/jwst#2662]
Running the `calwebb_spec3` pipeline on an association of MIRI MRS data results in an error in the `outlier_detection` step, specifically in the `blot_cube_build` module:
```
2018-10-03 16:33:10,707 - stpipe.Spec3Pipeline.outlier_detection - INFO - Information on Blotting
2018-10-03 16:33:10,707 - stpipe.Spec3Pipeline.outlier_detection - INFO - Working with instrument MIRI MIRIFUSHORT
2018-10-03 16:33:10,707 - stpipe.Spec3Pipeline.outlier_detection - INFO - shape of sky cube 52.000000 38.000000 929.000000
2018-10-03 16:33:10,707 - stpipe.Spec3Pipeline.outlier_detection - INFO - Instrument MIRI 
2018-10-03 16:33:10,708 - stpipe.Spec3Pipeline.outlier_detection - INFO - Channel 1
2018-10-03 16:33:10,708 - stpipe.Spec3Pipeline.outlier_detection - INFO - Sub-channel short
2018-10-03 16:33:10,708 - stpipe.Spec3Pipeline.outlier_detection - INFO - ROI size (spatial and wave) 0.150000 0.001000
2018-10-03 16:33:10,708 - stpipe.Spec3Pipeline.outlier_detection - INFO - Number of input models 2 
2018-10-03 16:33:12,170 - stpipe.Spec3Pipeline.outlier_detection - INFO - Blotting back cal_001_cal.fits
----------------------------------------------------------------------
ERROR RUNNING STEP 'Spec3Pipeline':
    too many values to unpack (expected 2)
Traceback (most recent call last):
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/stpipe/cmdline.py", line 297, in step_from_cmdline
    step.run(*positional)
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/stpipe/step.py", line 400, in run
    step_result = self.process(*args)
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/pipeline/calwebb_spec3.py", line 120, in process
    result = self.outlier_detection(result)
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/stpipe/step.py", line 400, in run
    step_result = self.process(*args)
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/outlier_detection/outlier_detection_step.py", line 166, in process
    step.do_detection()
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/outlier_detection/outlier_detection_ifu.py", line 143, in do_detection
    self.blot_median(median_model)
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/outlier_detection/outlier_detection_ifu.py", line 192, in blot_median
    blot_models = cubeblot.blot_images()
  File "/Users/bushouse/anaconda3/envs/myjwst/lib/python3.6/site-packages/jwst-0.12.0a0.dev138-py3.6-macosx-10.7-x86_64.egg/jwst/cube_build/blot_cube_build.py", line 221, in blot_images
    y, x = good_data
ValueError: too many values to unpack (expected 2)
```
Input data files and cfg files are available in /grp/jwst/ssb/bushouse/jwst_data/MIRI/MRS/kmurray/. The pipeline is being run using:
```
strun calwebb_spec3.cfg test_asn2.json
```
which uses the input files "cal_001_cal.fits" and "cal_002_cal.fits".

**Comment by jemorrison** [comment]
With PR#2598 I fixed a bug in the blotting routine. I believe this PR will fix this problem.
I can double check that and I will grab the data and run it with the cube_build that includes this fix to make sure.

**Comment by hbushouse** [comment]
You're right. When I run it with the updates from #2598 merged the error goes away and calspec3 runs to completion. Closing this issue.

**Comment by hbushouse** [comment]
Fixed by #2598
